assets/js/dist/sidebar.js

```diff
@@ -4487,7 +4488,22 @@ SelectMultiControl = AbstractControl.extend( {
     onRender : function() {
         _.each( this.getValues(), function( value, media ) {
             var $field = this.ui.input.filter( '[name^="' + media + '"]' );
-            $field.select2()
+
+            if (window[$field[0].id]) {
```

Comment:
Instead of just getting the 0 index item, it would be safer/better to do 
```js
var $fields = this.ui.input.filter( '[name^="' + media + '"]' );
$fields.forEach($field => {
  if ($field.id && window[$field.id]) {
    // Do your thing
  } else {
    $field.select2();
  }
});
```